src/api/hocrrenderer.cpp

```diff
@@ -319,7 +319,6 @@ char *TessBaseAPI::GetHOCRText(ETEXT_DESC *monitor, int page_number) {
             hocr_str << "\n        </span>";
             ++scnt;
           } else if (lstm_choice_mode == 2) {
-            tesseract::ChoiceIterator ci(*res_it);
```

Comment:
I think it would be better to keep this line and move the first declaration of `ci` into the `if` block.